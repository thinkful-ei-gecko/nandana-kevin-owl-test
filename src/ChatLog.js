import React from 'react';
import './ChatLog.css';

function ChatLog(props) {
  return (
    <div></div>
  )
}

export default ChatLog;