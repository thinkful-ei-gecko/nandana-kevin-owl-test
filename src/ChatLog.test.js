import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ChatLog from './ChatLog';

describe('<ChatLog />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ChatLog chatEvents={[]} participants={[]}/>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('chatlog renders as expected', () => {
    const tree = renderer.create(<ChatLog chatEvents={[]} participants={[]} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});