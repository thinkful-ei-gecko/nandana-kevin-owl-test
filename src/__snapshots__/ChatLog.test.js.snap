// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChatLog /> chatlog renders as expected 1`] = `
<div
  className="ChatLog"
>
  <h1>
    Chat Log
  </h1>
  Nobody!
</div>
`;
